saleor/graphql/product/types.py

```diff
@@ -202,6 +228,12 @@ class Meta:
         interfaces = [relay.Node]
         model = models.ProductVariant
 
+    @permission_required('product.manage_products')
+    def resolve_digital_content(self, info):
+        if hasattr(self, 'digital_content'):
```

Comment:
You would achieve the same result with: `return getattr(self, 'digital_content', None)`.